baseplate/sidecars/event_publisher.py

```diff
@@ -116,7 +116,9 @@ def serialize(self) -> SerializedBatch:
 
 publishCountTotal = Counter("event_publish_total", "total count of published events")
 publishLatency = Histogram("event_publish_latency", "latency for publishing a batch of events")
```

Comment:
Should this have a label indicating how many events were in the batch? (would increase cardinality but would also give some identifier why one call was [potentially] slower than another)